src/test/java/Homework21.java

```diff
@@ -0,0 +1,56 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest{
+    String newPlayListName = "NewPlayListName";
+    String currentPlayListName = "HW17";
+    @Test
+    public void renamePlaylist() throws InterruptedException {
+
+        login("dmytro.kravchuk@testpro.io", "Fr440003");
+        WebElement playList =  wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(String.format("//section[@id='playlists']//li/a[text()='%s']", currentPlayListName))));
+Thread.sleep(3000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.